adrs/5541831634/README.md

```diff
@@ -0,0 +1,157 @@
+# React project files and directories names conversion
+
+* **State:** Draft
+* **Created:** 2020-12-01
+* **Tags:** react
+
+## Context
+
+Across the ecosystem, we can find all sorts of files and directory naming
+conventions in React projects.
+
+The variety of conventions causes misalignment and shifts focus away from more
+important matters.
+
+Although there are no silver bullets for this topic, committing to a convention
+will reduce cognitive load by triggering pattern recognition, enabling engineers
+to focus their brainpower on their tasks.
+
+## Resolution
+
+* You **must** follow [NodeJS files and directories names conversion](./../3122196229/README.md)
+* You **must** use `<rootDir>/src/@types` to store TypeScript definition files.
```

Comment:
```suggestion
* You **must** use `<rootDir>/src/@types` to store global TypeScript definition files.
```

Maybe global isn't the right word... But we should draw distinction with `<fractalDir>/types`

Reply:
I think I spelled out the right term, it is a matter of understanding TypeScript lingos.

TypeScript Definitions files are the ones that use `*.d.ts` suffix, maybe suggest adding `(.d.ts)` to clarify.

Reply:
Good idea